server/lib/date-adjust.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+const fs = require("fs");
+
+// Write recent dates to initial-tweets.json
+// Specifically sync to not interfere with student functions
+module.exports = () => {
+  // Path to the initial tweets
+  const tweetsPath = "./server/data-files/initial-tweets.json";
+  // Read the tweets synchronously
+  const tweetsRead = JSON.parse(fs.readFileSync(tweetsPath, { encoding: "utf8" }));
+  // Iterate tweets and convert the previous time to now()...
+  // And subtract one day (1000ms * 60sec * 60min * 24hrs) * (2 - index)
+  tweetsRead.map((tweet, index) => tweet.created_at = Date.now() - (1000 * 60 * 60 * 24 * (2 - index)));
```

Comment:
What happens if we add more default tweets to `initial-tweets.json`? Wouldn't `2 - index` result in a negative number which would mean a date in the future?

Reply:
It would - I wrote that with the assumption that only 2 tweets would ever be presented as 'test data' - which I'm considering might be a bad practice.

I will look at a more future friendly method.